packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -68,18 +68,6 @@ The client-based rate-limiting framework can exist in one of three modes:
 | **Log per client**                          | Rate limiting is based on the [org-wide Rate Limit](/docs/reference/rate-limits/) values, but client-specific rate limiting violation information is logged as System Log events. |
```

Comment:
in the Enforce and log per client (recommended) row just above this one, you should add a link (using the words System Log as your link words) to the new client-based system log event section on the system log event page.